### ISSUE TITLE
Failed test in current NewPipeExtractor branch

### ISSUE THREAD

**Issue by jogacolhue** [TeamNewPipe/documentation#24]
Hello, I'm currently trying to debug some issues with the Extractor, I followed the guide to set the dev environment, but I'm having issues runing the tests, some of them are failing parsing Peertube's site. Can I ignore those results, or all the tests have to pass in order to start developing? 

Here's a picture of the testing results:

![pic1](https://user-images.githubusercontent.com/10107118/97396811-d49ae400-18b5-11eb-8dc0-bac09a0f18e2.png)

And here's a picture of the detailed error in the first test:

![pic2](https://user-images.githubusercontent.com/10107118/97396824-d8c70180-18b5-11eb-8e7f-92c3afc61403.png)

**Comment by Stypox** [comment]
Yes, you can ignore such results... Since streaming services keep changing things here and there, things like this may happen, it's not your fault.
